### DISCUSSION TITLE
FileNotFoundError: [WinError 3] The system cannot find the path specified: 'Y:\\Plots12'

### DISCUSSION THREAD

**Discussion by bl0atedgat0z** [swar/Swar-Chia-Plot-Manager#433] [Q&A]
I keep getting the 'Y:\\' (Double \\ like in the subject line) error in the log when I dont have that in my config.yaml.  I have removed the Swar directory and reinstalled Python with a reboot.

[12028] Failed to execute script chia
Traceback (most recent call last):
  File "pathlib.py", line 1273, in mkdir
FileNotFoundError: [WinError 3] The system cannot find the path specified: 'Y:\\Plots13'

During handling of the above exception, another exception occurred:

From my config.yaml in the Swar dir.
    temporary_directory: Z:\
    temporary2_directory:
    destination_directory:
      - Y:\Plots12
      - Y:\Plots13
Any help would be appreciated

**Comment by zyjdsg** [comment]
/
